eclass/qt5-build.eclass

```diff
@@ -655,7 +655,9 @@ qt5_base_configure() {
 
 		# disable undocumented X11-related flags, override in qtgui
 		# (not shown in ./configure -help output)
-		-no-xkb -no-xrender
+		-no-xkb
+		$([[ ${QT5_MINOR_VERSION} -lt 8 ]] || $([[ ${QT5_MINOR_VERSION} == 8
```

Comment:
- No idea...  It was probably the first thing that worked...  Adjusting to just one test (still seem to need the one '$(...)', or things seem to break, but that's probably to be expected).
- That was a silly mistake!  Using '-eq' now.
- Good point!  Changing.